packages/main/src/pages/static/contact-us/components/FaqSelected.jsx

```diff
@@ -1,11 +1,7 @@
 import ContactFormContainer from "./ContactFormContainer";
 
 const FaqSelected = () => {
-  return (
-    <div className="FAQselected">
-      <ContactFormContainer />
-    </div>
-  );
+  return <div className="FAQselected">{/* <ContactFormContainer /> */}</div>;
 };
```

Comment:
Your pr is still affecting the contact page. The code in red should be what is in that file. not the code in green